src/main/java/com/market/saessag/domain/user/dto/UserResponse.java

```diff
@@ -1,10 +1,11 @@
 package com.market.saessag.domain.user.dto;
 
+import lombok.Builder;
 import lombok.Getter;
 import lombok.Setter;
 
 @Getter
-@Setter
+@Builder
 public class UserResponse {
     private Long id;
```

Comment:
DTO의 필드 값이 변경될 일이 없다면 final 키워드를 사용하는 것이 어떨까요?
```suggestion
    private final Long id;
```

Reply:
좋은 것 같습니다! 상품 DTO도 변경될 일 없는 필드는 final 키워드 추가 해 보겠습니다